gdsfactory/components/mzi.py

```diff
@@ -37,6 +37,7 @@ def mzi(
     cross_section_x_top: Optional[CrossSectionSpec] = None,
     cross_section_x_bot: Optional[CrossSectionSpec] = None,
     mirror_bot: bool = False,
+    return_all_optical_ports: bool = False,
```

Comment:
How about naming this `add_optical_ports`?

another option is `add_all_ports` which would add all ports (optical, electrical, placement ...)

Reply:
Yes whatever makes most sense :)

Reply:
We just need to differentiate between two and bottom arm, for instance with prefix, in case the devices have the same port names